### PR TITLE
Nimble Setup

### PR DESCRIPTION
**Pull request by zah**

### CODE REVIEW

File: config.nims [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# begin Nimble config (version 1)
```
[the remark this answers is not in the document]
Comment (zah) [reply config.nims:1]: The `nimble setup` have to be executed once per repo (by a single developer who can then commit the result as it's done here)

[the remark this answers is not in the document]
Comment (arnetheduck) [reply config.nims:1]: no, I mean that config.nims has to be run for every dependency for every build - this costs disk accesses and time to instantiate a vm and so on - over time, we'll see a gradual slowdown of builds cause by this

[the remark this answers is not in the document]
Comment (zah) [reply config.nims:1]: That's not true. The config files of libraries are not loaded in projects that use the library. In other words, the configuration of the top-level project affects all modules that were imported as part of the build, no matter which library they originate from.

[the remark this answers is not in the document]
Comment (arnetheduck) [reply config.nims:1]: they're not? ok ... hm. that's going to be interesting then, for other reasons :)

**Pull request merged**